visidata/features/errors_guide.py

```diff
@@ -0,0 +1,29 @@
+from visidata import GuideSheet, vd
+
+class ErrorsGuide(GuideSheet):
+    guide_text='''# A guide to errors in VisiData
+
+There are two severity levels VisiData will use when communicating something has gone wrong.
+If in performing an operation it reaches a [:166]warning[/], the operation will continue.
+With [:202]errors[/], operations will stop.
+
+[:202]Errors[/] and/or [:166]warnings[/] can manifest in different ways. How they appear determines how they are folllowed up on.
+
+1. [:202]a phrase all in lowercase[/] [:166]appears in the sidebar[/]
+This is a hand-written message written for a known potential scenario. An example is *"invalid regex"* when the user passes an invalid regex string to a VisiData command.
+For more information on what condition triggered the message `git grep` for it within the VisiData repo.
+
+2. The first line of a stacktrace for an [:202]Exception[/] appears in the sidebar.
+Most `Exceptions` in VisiData are caught, and VisiData will continue running even if an operation has failed.
```

Comment:
Let's be more sparing with `code` within VisiData itself.  Keystrokes, longnames, CLI options--anything a user would actually type.  `Exception` is more of a dev word.

Reply:
Which word should I use instead of `Exception`? Just describe that they are looking at a traceback or stacktrace? I'm feeling conflicted, because they will bump into `Exception`s if they are interacting with the **Errors Sheet**.